pom.xml

```diff
@@ -371,7 +371,7 @@
 		<lucene.version>8.9.0</lucene.version>
 		<solr.version>8.9.0</solr.version>
 		<elasticsearch.version>7.15.2</elasticsearch.version>
-		<spring.version>5.3.30</spring.version>
```

Comment:
The spring version should probably match the spring version used in spring-boot. You can update spring-boot to 2.7.18 which uses spring 5.3.31. I think that that version also fixes the CVE.

Spring-boot version should probably be specified in the root pom. At the moment it's defined here: https://github.com/eclipse-rdf4j/rdf4j/blob/52b4106356a10584d63fb710d98654908c661e91/spring-components/pom.xml#L16

Reply:
good idea

Though probably not the only place, since the rdf4j-server.war also includes spring 5.x jars in the WEB-INF/lib, but not springboot (do we actually used Spring there ?)

Reply:
Spring is used in the server. Spring-boot is not used in either the server or workbench. But spring-boot is used in its own sub-module where there are some spring-boot related things and also an example app. 

Most people I know only use spring-boot now a days. Some might use a different framework, but I've not heard of anyone that has started a new project with just spring and not spring boot. So it's a good thing to have.

Reply:
I didn't know that we have our own jars in the web-inf for spring. That doesn't sound like a good idea.